examples/dgmulti_1d/elixir_euler_shu_osher_gauss_shock_capturing.jl

```diff
@@ -0,0 +1,93 @@
+    x0 = -4
+
+    rho_left = 27 / 7
+    v_left = 4 * sqrt(35) / 9
+    p_left = 31 / 3
+
+    # Replaced v_right = 0 to v_right = 0.1 to avoid positivity issues.
+    v_right = 0.1
+    p_right = 1.0
+
+    rho = ifelse(x[1] > x0, 1 + 1 / 5 * sin(5 * x[1]), rho_left)
+    v = ifelse(x[1] > x0, v_right, v_left)
+    p = ifelse(x[1] > x0, p_right, p_left)
+
+    return prim2cons(SVector(rho + 0.1 * rand(), v + 0.1 * rand(), p + 0.1 * rand()),
+                     equations)
+end
+
+initial_condition = initial_condition_shu_osher
+
+surface_flux = flux_lax_friedrichs
+volume_flux = flux_ranocha
+
+polydeg = 3
+basis = DGMultiBasis(Line(), polydeg, approximation_type = GaussSBP())
+
+indicator_sc = IndicatorHennemannGassner(equations, basis,
+                                         alpha_max = 0.5,
+                                         alpha_min = 0.001,
+                                         alpha_smooth = true,
+                                         variable = density_pressure)
+volume_integral = VolumeIntegralShockCapturingHG(indicator_sc;
+                                                 volume_flux_dg = volume_flux,
+                                                 volume_flux_fv = surface_flux)
+
+dg = DGMulti(basis,
+             surface_integral = SurfaceIntegralWeakForm(surface_flux),
+             volume_integral = volume_integral)
+
+boundary_condition = BoundaryConditionDirichlet(initial_condition)
+boundary_conditions = (; :entire_boundary => boundary_condition)
+
+###############################################################################
+#  setup the 1D mesh
+
+cells_per_dimension = (64,)
+mesh = DGMultiMesh(dg, cells_per_dimension,
+                   coordinates_min = (-5.0,), coordinates_max = (5.0,),
+                   periodicity = false)
+
+###############################################################################
+#  setup the semidiscretization and ODE problem
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition,
+                                    dg, boundary_conditions = boundary_conditions)
+
+tspan = (0.0, 2.0)
+ode = semidiscretize(semi, tspan)
+
+###############################################################################
+#  setup the callbacks
+
+# prints a summary of the simulation setup and resets the timers
+summary_callback = SummaryCallback()
+
+# analyse the solution in regular intervals and prints the results
+analysis_callback = AnalysisCallback(semi, interval = 100, uEltype = real(dg))
+
+# handles the re-calculation of the maximum Δt after each time step
+stepsize_callback = StepsizeCallback(cfl = 0.1)
+
+# collect all callbacks such that they can be passed to the ODE solver
+callbacks = CallbackSet(summary_callback, analysis_callback, stepsize_callback)
+
+# ###############################################################################
+# # run the simulation
+
+sol = solve(ode, SSPRK43(), adaptive = true, callback = callbacks, save_everystep = true)
```

Comment:
```suggestion
sol = solve(ode, SSPRK43(), adaptive = true, callback = callbacks, save_everystep = false)
```
We should not save every step in our standard examples (e.g., to save memory in CI runs).

Reply:
I guess the new test will look something like that
What do the entries `linf` and `l2` correspond to? 
```
@trixi_testset "elixir_euler_shu_osher_gauss_shock_capturing.jl " begin
    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_shu_osher_gauss_shock_capturing.jl"),
                        cells_per_dimension=(64,), tspan=(0.0, 1.0),
                        # division by sqrt(2.0) corresponds to normalization by the square root of the size of the domain
                        l2=[
                            7.853842541289665e-7,
                            9.609905503440606e-7,
                            2.832322219966481e-6,
                        ] ./ sqrt(2.0),
                        linf=[
                            1.5003758788711963e-6,
                            1.802998748523521e-6,
                            4.83599270806323e-6,
                        ])
    # Ensure that we do not have excessive memory allocations
    # (e.g., from type instabilities)
    let
        t = sol.t[end]
        u_ode = sol.u[end]
        du_ode = similar(u_ode)
        @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000
    end
end
```

Reply:
To obtain the values of `linf` and `l2` it is fine that you run the elixir locally, obtain the values from the local run by calling `analysis_callback(sol)`, and copy-past the values in the test (of course, after you manually double-checked that the results are expected).

Reply:
@ranocha, I have modified the example accordingly. The tests are passing on my local machine.